include/libtorrent/aux_/portmap.hpp

```diff
@@ -66,6 +66,31 @@ namespace libtorrent { namespace aux {
 	protected:
 		~portmap_callback() {}
 	};
+	struct base_mapping_t
```

Comment:
would you mind adding a blank line in between here, and rename ``base_mapping_t`` to just ``base_mapping``